docs/docusaurus.config.ts

```diff
@@ -8,14 +8,14 @@ const config: Config = {
   favicon: 'img/favicon.ico',
 
   // Set the production url of your site here
-  url: 'https://your-docusaurus-site.example.com',
+  url: 'https://aether.green',
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
   baseUrl: '/',
 
   // GitHub pages deployment config.
   // If you aren't using GitHub pages, you don't need these.
-  organizationName: 're-cinq', // Usually your GitHub org/user name.
+  organizationName: 're:cinq', // Usually your GitHub org/user name.
```

Comment:
Maybe remove these since we aren't using GitHub pages, or maybe put it back to a `-` since that matches our repo org if we were to shift in the future :shrug: